docs/content/latest/yugabyte-platform/create-deployments/async-replication-platform.md

```diff
@@ -11,12 +11,110 @@ menu:
+
+4. Provide the name for your replication.
+
+5. Select the target universe.
+
+6. Click **Next: Select Tables**.
+
+7. From a list of common tables between source and target universes, select the tables you want to include in the replication and then click **Create Replication**, as per the following illustration:<br><br>
+
+   ![Create Replication](/images/yp/asynch-replication-3.png)
+
+### How to configure replication
+
+You can configure an existing replication as follows:
+
+1. Open the Yugabyte Platform UI and navigate to **Universes**.
+
+2. Select the universe whose existing replication you want to modify and then navigate to **Replication**, as per the following illustration:<br><br>
+
+   ![Replication](/images/yp/asynch-replication-1.png)<br><br>
+
+3. Click **Configure Replication** and perform steps 4 through 7 from [How to set up replication](#set-up).
+
+### How to view, manage, and monitor replication
+
+To view and manage an existing replication, as well as configure monitoring, click the replication name to open the details page shown the following illustration:
+
+<br>
+
+![Replication Details](/images/yp/asynch-replication-4.png)
+
+This page allows you to do the following:
+
+- View the replication details.
+
+- View and modify the list of tables included in the replication, as follows:
+
+  - Select **Tables**, as per the following illustration:<br><br>
+
+    ![Tables](/images/yp/asynch-replication-7.png)<br><br>
+
+  - Click **Modify Tables**. 
+
+  - Use the **Add tables to the replication** dialog to change the table selection, as per the following illustration:<br><br>
+
+    ![Change Tables](/images/yp/asynch-replication-8.png)<br><br>
+
+    The following illustration shows the **Add tables to the replication** dialog after modifications:<br><br>
+
+    ![Change Tables](/images/yp/asynch-replication-9.png)<br><br>
+
+- Configure the replication, as follows:
+
+  - Click **Actions > Edit replication configuration**.
+
+  - Make changes using the **Edit cluster replication** dialog shown in the following illustration:<br><br>
+
+    ![Edit Replication](/images/yp/asynch-replication-5.png)<br><br>
+
+- Set up monitoring by configuring alerts, as follows:
+
+  - Click **Configure Alert**.
+
+  - Use the **Configure Replication Alert** dialog to enable or disable alert issued when the replication lag exceeds the specified threshold, as per the following illustration:<br><br>
+
+    ![Alert](/images/yp/asynch-replication-6.png)<br><br>
+
+- Pause the replication process by clicking **Pause Replication**. This is useful when performing maintainance.
+
+- Delete the universe replication by clicking **Actions > Delete replication**.
+
+## Use the REST API
+
+You may chose to use the API to manage universes. You can call the following REST API endpoint on your Yugabyte Platform instance for the source universe and the target universe involved in the asynchronous replication between two data sources:
```

Comment:
"You may choose to" typo

'You can call the following REST API endpoint' - don't quite follow. Does this PUT  command set up the replication? The rest of this section just seems to describe how to find your UUIDs.

Reply:
This section is not needed. I'm deleting it.